types/echarts/index.d.ts

```diff
@@ -1582,7 +1582,8 @@ declare namespace echarts {
                 /**
                  * @todo describe
                  */
-                interface Label extends TextStyleWithRich {
+                interface Label extends Omit<TextStyleWithRich,'color'> {
```

Comment:
Does `TextStyleWithRich` have a `color` property to omit?

Reply:
interface TextStyleWithRich extends TextStyle，interface TextStyle extends BaseTextStyle，interface BaseTextStyle has a **color** property

Reply:
Oops, I was looking at `BaseTextStyleWithRich` instead of `TextStyleWithRich`.